src/models/link.es6.js

```diff
@@ -82,23 +82,6 @@ class Link extends Base {
     );
   }
 
-  get thumbnail () {
-    var props = this.props;
-    var thumbnailSrc = '/img/default.gif';
-
-    if (props.thumbnail) {
-      if (props.thumbnail === 'default' || props.thumbnail === 'self' || props.thumbnail === 'nsfw') {
```

Comment:
It would be a 404 unless you explicltly returned `undefined`, which would cause it to not load a thumbnail at all instead of attempting to load a `/default` or `/self` or `/nsfw` route.